### PR TITLE
don't repeat yourself

### PR DESCRIPTION
**Pull request by benaryorg**

### CODE REVIEW

File: src/qml/patchmanager.cpp [hunk 1]
```diff
@@ -115,9 +115,7 @@ void PatchManager::restartServices()
     }
 
     if (m_homescreenNeedRestart) {
-        QStringList arguments;
-        arguments << "--user" << "restart" << "lipstick.service";
-        QProcess::startDetached("systemctl", arguments);
+        restartLipstick();
```
Comment (SfietKonstantin) [inline src/qml/patchmanager.cpp]: You can also remove the second if (m_homescreenNeedRestart) that is following this if

**Comment by SfietKonstantin** [comment]
Interesting error ... Only one comment and LGTM

File: src/qml/patchmanager.cpp (hunk 1 above)
Comment (CODeRUS) [inline src/qml/patchmanager.cpp]: I'am voting for rewriting it to use D-Bus calls instead of QProcess.

Comment (SfietKonstantin) [inline src/qml/patchmanager.cpp]: Is there a DBus call to restart lipstick ? Or do you want to go with a call to systemd ?

Comment (CODeRUS) [inline src/qml/patchmanager.cpp]: call to systemd. i'll look into this later today.

Comment (SfietKonstantin) [inline src/qml/patchmanager.cpp]: Thanks, waiting for PR

**Comment by benaryorg** [comment]
Just ping me when it's time to close this PR.…

**Comment by CODeRUS** [comment]
you can close it. but today i have some weird problems with SDK.

**Comment by CODeRUS** [comment]
Closed because of https://github.com/sailfishos-patches/patchmanager/pull/8

**Pull request closed without merging**